flavour/azure/Dockerfile.flavour

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/azure-cli:latest
+FROM mcr.microsoft.com/azure-cli:2.63.0
```

Comment:
If we do not want to change our setup we should consider building our own image from scratch.

Reply:
I'd rather use the official one. So we don't have to think about updating more than the things we develop.